packages/api/amplify_api/lib/src/method_channel_api.dart

```diff
@@ -333,106 +333,130 @@ class AmplifyAPIMethodChannel extends AmplifyAPI {
   }
 
   @override
-  CancelableOperation<AWSStreamedHttpResponse> get(
+  AWSHttpOperation get(
     String path, {
     Map<String, String>? headers,
     Map<String, String>? queryParameters,
     String? apiName,
   }) {
-    return _restFunctionHelper(
-      methodName: 'get',
-      path: path,
-      headers: headers,
-      queryParameters: queryParameters,
-      apiName: apiName,
+    return AWSHttpOperation(
+      _restFunctionHelper(
+        methodName: 'get',
+        path: path,
+        headers: headers,
+        queryParameters: queryParameters,
+        apiName: apiName,
+      ),
+      requestProgress: const Stream.empty(),
+      responseProgress: const Stream.empty(),
```

Comment:
The function changes in this file all include these lines. They are not correct and should be replaced with the proper streams.

Reply:
This file will be deleted soon so it's not a big deal, but yeah this is how you could handle it, or probably better would be `Stream.error` since `Stream.empty` just creates a Stream which immediately completes which could be misleading in this case since that is also how progress completion is conveyed typically.